localisation/fr-fr/docs/01-monkey-app.md

```diff
@@ -0,0 +1,164 @@
+    ```
+
+1. Entrez le prompt suivant à GitHub Copilot pour générer un issue sur le dépôt.
+
+    ```text
+    Create a new GitHub issue in my repository titled 'Implement Monkey Console Application' with the following requirements:
+    
+    - Create a C# console app that can list all available monkeys, get details for a specific monkey by name, and pick a random monkey.
+    - The app should use a Monkey model class and include ASCII art for visual appeal.
+    - Add appropriate labels like 'enhancement' and 'good first issue'.
+    - Add some details about how we may go about implementing this and a checklist for what we will need to do.
+    ```
+
+1. Assignez `@Copilot` à l'issue et observez ce qui se passe.
+
+## Démarrer le Serveur MCP &ndash; Monkey MCP
+
+1. Ouvrez la Palette de Commandes en tapant `F1` ou `Ctrl`+`Shift`+`P` sur Windows ou `Cmd`+`Shift`+`P` sur Mac OS, et recherchez `MCP: List Servers`.
+1. Assurez-vous que le serveur MCP `github` fonctionne.
+1. Choisissez `monkeymcp` puis cliquez sur `Start Server`.
+
+## Développer l'Application Monkey avec GitHub Copilot et les Serveurs MCP
+
+1. Entrez le prompt suivant pour obtenir la liste des singes.
+
+    ```text
+    Get me a list of monkeys that are available and display them in a table with their details.
+    ```
+
+1. Entrez le prompt suivant pour obtenir une idée du modèle de données pour un singe.
+
+    ```text
+    What would a data model look like for this structure?
+    ```
+
+1. Entrez le prompt suivant pour créer un fichier pour la classe du modèle de données.
+
+    ```text
+    Let's create a new file for this class.
+    ```
+
+1. Entrez le prompt suivant pour créer une classe `MonkeyHelper`.
+
+    ```text
+    Let's create a new class called MonkeyHelper that is static. It should manage a collection of monkey data. Include methods to get all monkeys, get a random monkey, find a monkey by name, and track access count to when a random monkey is picked. The data for the monkeys should come from the Monkey MCP server that we just got.
+    ```
+
+1. Entrez le prompt suivant pour mettre à jour l'application console.
+
+    ```text
+    Let's update the app now to have a nice menu with the following options that will call into that `MonkeyHelper`.
+    
+    1. List all monkeys
+    2. Get details for a specific monkey by name
+    3. Get a random monkey
+    4. Exit app
+
+    Also display some funny ASCII art randomly.
+    ```
+
+1. Entrez le prompt suivant à GitHub Copilot pour pousser l'application console mise à jour.
+
+    ```text
+    Push the current changes to the `mymonkeyapp` branch of the repository.
+    With this branch, create a PR against the `main` branch.
+    Connect this PR to the issue created before.
+    Then, merge this PR and close the issue.
+    ```
+
+---
+
+OK. Vous avez terminé l'étape "Développement d'Application Monkey avec MCP". Passons à [ÉTAPE 02 : Serveur MCP](./02-mcp-server.md).
```

Comment:
Make sure that the disclaimer should be included at the end of the localised document.